locals.tf

```diff
@@ -52,6 +54,7 @@ locals {
   feature_gates       = var.feature_gates
   admission_plugins   = var.admission_plugins
   region              = var.region
+  zone                = var.zone
```

Comment:
zone should not be top level local as it is only settable on node pool and not control plane

Reply:
zone should be below in `node_pools_defaults_defaults`